routes/create.js

```diff
@@ -1,18 +1,51 @@
 const express = require("express");
+const Story = require("../models/Story");
+const User = require("../models/User");
+const Chapter = require("../models/Chapter");
 const router = express.Router();
 
 // @desc Create story page
 // @route POST /create
 // @access Private
 router.post("/", (req, res) => {
-  res.send("it should take book details"); // this should return back a storyID which will be used to create the chapter 
+  router.post("/", async(req, res) => {
+    try {
+              await Story.create(req.body, async function (err, story) {
+                  await User.updateOne(
+                      { _id: req.params.user },
```

Comment:
We don't have users in params so we can't extract users from the route itself. Use passport Auth like I have explained earlier to make the user available.